services/users.js

```diff
@@ -15,7 +16,7 @@ async function signup (firstName, lastName, email) {
   }
   const secret = uuidv4()
   UserModel.create(firstName, lastName, email, secret)
```

Comment:
[nit] unrelated, but probably should `await UserModel.create` before returning to user, otherwise they might try to add a friend before their account exists.